subprojects/docs/src/docs/userguide/custom_tasks.adoc

```diff
@@ -331,7 +331,7 @@ This way when the task's results are loaded from cache, the next execution can a
 
 However, if the state files are non-relocatable, then they can't be shared via the build cache.
 Indeed, when the task is loaded from cache, any such state files must be cleaned up to prevent stale state from confusing the tool during the next execution.
-Gradle can ensure such stale files are removed if they are declared via `link:{javadocPath}/org/gradle/api/tasks/TaskLocalState.html#register-java.lang.Object++...++-[task.localState.register()]` or if a property is marked with the `@link:{javadocPath}/org/gradle/api/tasks/LocalState.html[LocalState]` annotation.
+Gradle can ensure such stale files are removed if they are declared via `link:{javadocPath}/org/gradle/api/tasks/TaskLocalState.html#register-java.lang.Object++...++-[task.localState.register()]` or if a property is marked with the `link:{javadocPath}/org/gradle/api/tasks/LocalState.html[@LocalState]` annotation.
```

Comment:
Based on the discussion in the other thread, I would leave the backticks out for consistency with the other API links in this chapter. And I'll have to review my own use of them. I was clearly not paying attention.